corporate/lib/stripe.py

```diff
@@ -2973,8 +2972,9 @@ def get_customer(self) -> Optional[Customer]:
 
     @override
     def get_email(self) -> str:
-        # BUG: This is an email for the whole server. We probably
-        # need a separable field here.
+        if self.remote_billing_user is not None:
+            return self.remote_billing_user.email
```

Comment:
I think this can be `assert self.remote_billing_user is not None` -- we shouldn't need the fallback.